e2e/dockerutil/dockerutil.go

```diff
@@ -24,7 +24,8 @@ func GetTestContainers(ctx context.Context, t *testing.T, dc *dockerclient.Clien
 		Filters: filters.NewArgs(
 			// see: https://github.com/strangelove-ventures/interchaintest/blob/0bdc194c2aa11aa32479f32b19e1c50304301981/internal/dockerutil/setup.go#L31-L36
 			// for the label needed to identify test containers.
-			filters.Arg("label", testLabel+"="+t.Name()),
+			// TODO: Add handle test name
```

Comment:
Currently, because we are using a single container for each chain from start to finish of the test, I am fixing the label here. Maybe we'll have a better way to handle it in the future.